chainer/links/normalization/decorrelated_batch_normalization.py

```diff
@@ -0,0 +1,133 @@
+import numpy
+
+from chainer import configuration
+from chainer import functions
+from chainer import link
+from chainer.utils import argument
+
+
+class DecorrelatedBatchNormalization(link.Link):
+
+    """Decorrelated batch normalization layer.
+
+    This link wraps the
+    :func:`~chainer.functions.decorrelated_batch_normalization` and
+    :func:`~chainer.functions.fixed_decorrelated_batch_normalization`
+    functions. It works on outputs of linear or convolution functions.
+
+    It runs in three modes: training mode, fine-tuning mode, and testing mode.
+
+    In training mode, it normalizes the input by *batch statistics*. It also
+    maintains approximated population statistics by moving averages, which can
+    be used for instant evaluation in testing mode.
+
+    In fine-tuning mode, it accumulates the input to compute *population
+    statistics*. In order to correctly compute the population statistics, a
+    user must use this mode to feed mini-batches running through whole training
+    dataset.
+
+    In testing mode, it uses pre-computed population statistics to normalize
+    the input variable. The population statistics is approximated if it is
+    computed by training mode, or accurate if it is correctly computed by
+    fine-tuning mode.
+
+    Args:
+        size (int or tuple of ints): Size (or shape) of channel
+            dimensions.
+        groups (int): Number of groups to use for group whitening.
+        decay (float): Decay rate of moving average
+            which is used during training.
+        eps (float): Epsilon value for numerical stability.
+        dtype (numpy.dtype): Type to use in computing.
+
+    See: `Decorrelated Batch Normalization <https://arxiv.org/abs/1804.08450>`_
+
+    .. seealso::
+       :func:`~chainer.functions.decorrelated_batch_normalization`,
+       :func:`~chainer.functions.fixed_decorrelated_batch_normalization`
+
+    Attributes:
+        avg_mean (:ref:`ndarray`): Population mean.
+        avg_projection (:ref:`ndarray): Population
```

Comment:
Ops, missing a closing backtick.
```suggestion
        avg_projection (:ref:`ndarray`): Population
```

Reply:
Thank you